### DISCUSSION TITLE
Migrate other plugins to this repo

### DISCUSSION THREAD

**Discussion by zhan9san** [ansible-community/molecule-plugins#8] [General]
@ssbarnea 

Do we have a plan to move other plugins like docker to this repo?

molecule-docker CI jobs failed several days ago.

https://github.com/ansible-community/molecule-docker/actions

I am wondering if it's valuable to fix it in molecule-docker repo or migrate it to this repo directly.

**Comment by ssbarnea** [comment]
YES, is just that I was too busy with other things. Maybe I will find some time this weekend to do it.
